autoload/fakeclip.vim

```diff
@@ -37,6 +37,10 @@ else
   let s:PLATFORM = 'unknown'
 endif
 
+" oclip
+if g:clipboard_platform == 'oclip'
```

Comment:
It's an error to refer undefined variable, for example:
https://travis-ci.org/kana/vim-fakeclip/builds/631167292?utm_source=github_status&utm_medium=notification
> ```
> -------- Testing a Vim plugin
> t/clipboard-io.vim .... 
> # -----------------------------------------------------------------------------
> # /home/travis/build/kana/vim-fakeclip/autoload/fakeclip.vim, line 41
> # Vim(if):E121: Undefined variable: g:clipboard_platform
> ```

Reply:
Put this `if` block above line 26, for example:

```vim
if g:clipboard_platform == 'oclip'
  let s:PLATFORM = 'oclip'
elseif has('macunix') || system('uname') =~? '^darwin'
  let s:PLATFORM = 'Mac'
...
```

Since there are many supported platform, it would be better to keep the order of platform detection code to match its priority.

Reply:
Use `g:fakeclip_platform` instead of `g:clipboard_platform`, because this plugin uses `g:fakeclip_` as a prefix for global variables.

Reply:
Hmm, there are TWO types of values for `s:PLATFORM`.  One uses commands specific to OS (`'mac'`, `'wsl'`, `'cygwin'` and `'x'`), another uses commands not specific to OS (`'lemonade'`). oclip corresponds to the latter.  It would be better to automatically detect oclip (like lemonade) in addition to `g:clipboard_platform`.

So that I prefer rewriting the whole of platform detection code like this:

```vim
if index(['lemonade', 'oclip'], get(g:, 'fakeclip_platform', '')) >= 0
  let s:PLATFORM = g:fakeclip_platform
elseif has('macunix') || system('uname') =~? '^darwin'
  let s:PLATFORM = 'mac'
[...]
elseif executable('lemonade')
  let s:PLATFORM = 'lemonade'
elseif executable('oclip')
  let s:PLATFORM = 'oclip'
else
  let s:PLATFORM = 'unknown'
endif
```